content/08.methods.md

```diff
@@ -24,15 +24,18 @@ Manubot allowed us to automate this process, using the Git commit hash as a rand
 
 #### Version 2.0
 
-We continued using the open repository on the GitHub version control platform ([`greenelab/deep-review`](https://github.com/greenelab/deep-review)) [@url:https://github.com/greenelab/deep-review], which was established to write the Version 1.0 manuscript. {% for v2, authors in manubot.authors|groupby('v2') %}{% if v2|length %}{{ v2 }}: {% for author in authors %}{{author.name }}{%- if not loop.last -%},  {%- endif -%}{% endfor %}. {% endif %}{% endfor %}
+We continued using the open repository on the GitHub version control platform ([`greenelab/deep-review`](https://github.com/greenelab/deep-review)) [@url:https://github.com/greenelab/deep-review], which was established to write the version 1.0 manuscript.
+{% for v2, authors in manubot.authors|groupby('v2') %}
+  {% if v2|length %}{{ v2 }}: {% for author in authors %}{{author.name }}{% if not loop.last %}, {% endif %}{% endfor %}. {% endif %}
```

Comment:
It's hard to see with the added newlines, but removing some `-` here added spaces between the authors:
![image](https://user-images.githubusercontent.com/3532898/76710613-f0f76100-66d6-11ea-91bb-3e38b21450b4.png)